.github/workflows/tf-nightly.yml

```diff
@@ -58,6 +73,7 @@ jobs:
           tfsec --version
 
       - name: Test Terraform
+        if: ${{ ! inputs.use-custom-aws-creds }}
```

Comment:
not clear on this one, why no test terraform without custom creds?

Reply:
Ahh good catch that was a leftover line that was meant to be removed. Fixed.